packages/components/src/Spinner/Spinner.mdx

```diff
@@ -0,0 +1,53 @@
+---
+name: Spinner
+menu: Components
+route: /components/spinner
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Spinner } from ".";
+import { Text } from "../Text";
+
+# Spinner
+
+<ComponentStatus stage="pre" responsive="no" accessible="yes" />
+
+Spinners are used to visualize an undetermined load time for a section of a page
+and prevent double clicking of a UI element by overlaying it while its action is
+processing.
+
+The default spinner should be used to indicate loading of a view
+
+```ts
+import { Spinner } from "@jobber/components/Spinner";
+```
+
+<Playground>
+  <Spinner />
+</Playground>
+
```

Comment:
I'd like to add this section, it's part of our new docs template but this particular component PR is based on some older work so it's not captured here:

## Design & Usage Guidelines

Use `Spinner` to indicate loading of content where the actual amount of loading time or progress is unknown.

### Related Components

If the amount of progress or loading time is known, use [ProgressBar](https://atlantis.getjobber.com/components/progress-bar) to give the user better context around the process.